docs/guides/unity-catalog.md

```diff
@@ -145,12 +145,21 @@ resource "aws_s3_bucket" "metastore" {
   })
 }
 
-resource "aws_s3_bucket_public_access_block" "root_storage_bucket" {
+resource "aws_s3_bucket_public_access_block" "metastore" {
   bucket             = aws_s3_bucket.metastore.id
   ignore_public_acls = true
   depends_on         = [aws_s3_bucket.metastore]
 }
 
+resource "aws_s3_bucket_public_access_block" "metastore" {
```

Comment:
here we've got two `aws_s3_bucket_public_access_block.metastore` resources. make it one again :)